test/evergreen.yml

```diff
@@ -1638,6 +1662,105 @@ tasks:
               cp -rf WT_TEST WT_TEST_$file
             done
 
+  - name: check-test-long-1
+    tags: ["check-long-test"]
+    commands: 
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          posix_configure_flags: ac_cv_func_ftruncate=no
```

Comment:
We have existing `compile wiredtiger no linux ftruncate` function. It may worth to confirm if this configure flag setting achieves the same effect as that of the function.